src/WebJobs.Extensions/WebJobs.Extensions.csproj

```diff
@@ -21,7 +21,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.31" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.32" />
```

Comment:
I am not sure if I need to update `Microsoft.Azure.WebJobs` ion other extensions in this repository.

Reply:
Might as well